### PR TITLE
165 fix actions pot matches

### LINKED ISSUE #165: fix actions and reducers for potential matches to redux
should dispatch action when the user clicks like or dislike. 
also map state to props

### PR DESCRIPTION
**Pull request by davidnicho**
closes #165 
It has all gone to pot! except for when it has an action to go with it.
not to master this time...

### CODE REVIEW

File: client/actions/potMatches.js [hunk 1]
```diff
@@ -5,37 +5,36 @@ export const POTENTIAL_MATCHES_SUCCESS = 'POTENTIAL_MATCHES_SUCCESS'
...
   }
 }
 
-export function potentialMatches (users) {
+export function fetchPotMatches(userId) {
   return dispatch => {
     dispatch(pendingPotentialMatches())
-    request
-      .get(`/api/v1/users/${users}`)
+    request.get(`/api/v1/users/${userId}/matches`)
```
Comment (tamari-gray) [inline client/actions/potMatches.js]: can you please change route to `/api/v1/users/${userId}/pot

File: client/components/PotMatches.jsx [hunk 2]
```diff
@@ -3,35 +3,21 @@ import { connect } from 'react-redux'
...
 
-  handleSubmit = () => this.setState({
-    profileId: '',
-    name: '',
-    description: '',
-    languages: []
-  })
-
   render () {
     const testarray = ['4', 'strings', 'another', 'string']
```
Comment (tamari-gray) [inline client/components/PotMatches.jsx]: can this be deleted please

File: client/components/PotMatches.jsx [hunk 3]
```diff
@@ -3,35 +3,21 @@ import { connect } from 'react-redux'
...
 
 class PotMatches extends React.Component {
-  state = {
-    profileId: '',
-    name: '',
-    description: '',
-    languages: []
-  }
-
   handleChange = (e, { id, user }) => this.setState({ [id]: user })
```
Comment (tamari-gray) [inline client/components/PotMatches.jsx]: dont need this function

Comment (davidnicho) [reply client/components/PotMatches.jsx]: done

**Comment by tamari-gray** [comment]
nice!

**Pull request merged**